tools/docker/builder/openwrt/Dockerfile

```diff
@@ -67,39 +67,39 @@ WORKDIR /home/openwrt/openwrt
 COPY --chown=openwrt:openwrt profiles_feeds/ /home/openwrt/openwrt/profiles_feeds
 
 RUN mkdir -p files/etc \
-#   We need to keep the hashes in the firmware, to later know if an upgrade is needed:
+    #   We need to keep the hashes in the firmware, to later know if an upgrade is needed:
     && printf '%s=%s\n' "OPENWRT_REPOSITORY" "$OPENWRT_REPOSITORY" >> files/etc/prplwrt-version \
     && printf '%s=%s\n' "OPENWRT_VERSION" "$OPENWRT_VERSION" >> files/etc/prplwrt-version \
     && if [ "$TARGET_PROFILE" = DEVICE_NETGEAR_RAX40 ] ; then \
-#       Add prplmesh to the list of packages of the profile:
-        sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/intel_mips.yml \
-        && yq write --inplace profiles/intel_mips.yml feeds -f profiles_feeds/netgear-rax40.yml \
-        && ./scripts/gen_config.py intel_mips \
-#       Installing intel feed doesn't correctly regenerate kernel .package-info
-#       force regeneration by removing it
-        && rm -rf tmp \
-#       For some reason we have to run gen_config a second time to get a correct .config:
-        && ./scripts/gen_config.py intel_mips \
-#       make sure intel's bridge-utils is the only one that gets installed:
-        && rm -rf ./package/feeds/packages/bridge-utils \
-        && scripts/feeds install -p feed_bridge_utils bridge-utils \
-        && cat profiles_feeds/netgear-rax40.yml >> files/etc/prplwrt-version ;\
+    #       Add prplmesh to the list of packages of the profile:
+    sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/intel_mips.yml \
```

Comment:
Why do you remove the indentation here? It *was* properly indented inside the condition...

Reply:
This is the default docker formatter in vscode, so apparently it wasn't properly indented according to its default rules.

Reply:
I reverted this change, but note that it is *against* the formatting rules so if someone will re-format this file it will be done again.